MQTicketService/Dockerfile

```diff
@@ -10,8 +10,14 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
-
-FROM docker.io/library/maven:3.6-openjdk-11 AS build
+#
+# Specify base image and platform for the build
+ARG imageName=icr.io/ibm-messaging/mq:latest
```

Comment:
Both are base images, should rename the ARGs so that it is more meaningful. eg. baseImageStep1, and baseImageStep2

Reply:
or baseImageBuildStage, baseImageRunStage